### PR TITLE
Cleanup working with `dnf.conf` and `yum.conf`

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

**Comment by sergiomb2** [comment]
Hi , I think this PR is a problem , I have to check it better to understand what is the goal , but I should have configurations that works with dnf and with yum .

File: mock/py/mockbuild/util.py [hunk 1]
```diff
@@ -1698,7 +1700,7 @@ def add_local_repo(config_opts, baseurl, repoid=None, bootstrap=None):
 best=1
 """.format(repoid=repoid, baseurl=baseurl)
 
-    config_opts['yum.conf'] += localyumrepo
+    config_opts['{0}.conf'.format(config_opts['package_manager'])] += localyumrepo
```
Comment (sergiomb2) [inline mock/py/mockbuild/util.py]: here , it beaks configuration if we want use dnf when we have yum.conf , for example epel7 , I guess

Comment (praiskup) [reply mock/py/mockbuild/util.py]: Indeed.  Can you please consider #486, would that help?

Comment (praiskup) [reply mock/py/mockbuild/util.py]: That said, it should work just fine if `package_manager` is in sync with `{yum,conf}.conf`.  It should
be the case in the latest mock-core-configs.

Comment (sergiomb2) [reply mock/py/mockbuild/util.py]: why not revert these commits ? yum.conf for all ! :)

Comment (sergiomb2) [reply mock/py/mockbuild/util.py]: what are the advantages, perspectives of having dnf.conf ?

**Pull request merged**